### PR TITLE
Document panther standard fields and example Athena queries

### PR DESCRIPTION
**Pull request by rleighton**
## Changes
Added files on standard panther fields and example queries.
Renamed root doc in historical search from untitled.md to README.md

### CODE REVIEW

File: docs/gitbook/historical-search/example-queries.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
+# Example queries
+
+Please note that all queries should be qualified with partition columns: year, month, day, hour for performance reasons.
+
+## Did this IP address have any activity in my network (and in what logs)?
+
+```
```
Comment (austinbyers) [inline docs/gitbook/historical-search/example-queries.md]: You can mark the code block as sql with ` ```sql `

```sql

SELECT p_log_type, count(1) as row_count
FROM panther_views.all_logs
WHERE ...
```

File: docs/gitbook/historical-search/panther-fields.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+# Panther standard fields
+
+Panther log processing appends some fields to all log records. This fields provide
```
Comment (austinbyers) [inline docs/gitbook/historical-search/panther-fields.md]: Typo: These *fields provide

File: docs/gitbook/historical-search/panther-fields.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+standard names for attributes over all data sources allowing within and between
+data source correlation of data.
```
Comment (austinbyers) [inline docs/gitbook/historical-search/panther-fields.md]: > allowing within and between data source correlation of data

Not sure what this means. Is it "allowing data correlation across multiple sources" ?

File: docs/gitbook/historical-search/example-queries.md [hunk 4]
```diff
@@ -0,0 +1,209 @@
+# Example queries
+
+Please note that all queries should be qualified with partition columns: year, month, day, hour for performance reasons.
```
Comment (austinbyers) [inline docs/gitbook/historical-search/example-queries.md]: Nit: Keep the list separate

> Please note that all queries should be qualified with partition columns (year, month, day, hour) for performance reasons.

or

> Please note that all queries should be qualified with partition columns for performance reasons: year, month, day, hour

File: docs/gitbook/historical-search/panther-fields.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+All appended standard fields begin with "p\_".
+
+# Required fields
+
+## p_log_type string
```
Comment (austinbyers) [inline docs/gitbook/historical-search/panther-fields.md]: Consider monospacing the field name:

## `p_log_type` string

File: docs/gitbook/historical-search/panther-fields.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+```
+
+From this information you can then explore the particular logs where activity is indicated.
+
+# Standard fields can be used in rules
+
+The Panther standard fields can be used in rules. For example, this rule triggers when any
+GuardDuty alert is on a resource tagged as 'critical':
+
+![](../.gitbook/assets/PantherStandardFieldRule.png)
```
Comment (austinbyers) [inline docs/gitbook/historical-search/panther-fields.md]: The first section `[]` is the accessibility text used when the image can't be loaded or a user has a screen-reader. Best practice is to put in a short description of the image, e.g.

`Panther Rule With Fields`

File: docs/gitbook/SUMMARY.md [hunk 7]
```diff
@@ -37,7 +37,7 @@
 
 ## Historical Search
 
-- [Background](historical-search/untitled.md)
```
Comment (jacknagz) [inline docs/gitbook/SUMMARY.md]: The new pages created in this PR have to be added at the same list level here, or else they won't show up on the navigation

File: docs/gitbook/historical-search/example-queries.md (hunk 1 above)
Comment (jacknagz) [reply docs/gitbook/historical-search/example-queries.md]: We can get rid of the leading/extra newlines too around the code block (nit)

File: docs/gitbook/historical-search/panther-fields.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
+# Panther standard fields
+
+Panther log processing appends some fields to all log records. This fields provide
+standard names for attributes over all data sources allowing within and between
+data source correlation of data.
+
+All appended standard fields begin with "p\_".
+
+# Required fields
```
Comment (jacknagz) [inline docs/gitbook/historical-search/panther-fields.md]: I think these fields would be better suited for a table than separate headers

File: docs/gitbook/SUMMARY.md (hunk 7 above)
Comment (rleighton) [reply docs/gitbook/SUMMARY.md]: Good to know. Fixed.

File: docs/gitbook/historical-search/panther-fields.md (hunk 8 above)
Comment (rleighton) [reply docs/gitbook/historical-search/panther-fields.md]: Definitely. Excellent suggestion. Done.

File: docs/gitbook/historical-search/example-queries.md (hunk 1 above)
Comment (rleighton) [reply docs/gitbook/historical-search/example-queries.md]: The sql tag highlights reserved words! Very nice. Fixed along with extra spacing.

File: docs/gitbook/historical-search/panther-fields.md (hunk 3 above)
Comment (rleighton) [reply docs/gitbook/historical-search/panther-fields.md]: I elaborated. Let me know if that helped.

File: docs/gitbook/historical-search/example-queries.md (hunk 4 above)
Comment (rleighton) [reply docs/gitbook/historical-search/example-queries.md]: Good catch, fixed.

File: docs/gitbook/historical-search/panther-fields.md (hunk 5 above)
Comment (rleighton) [reply docs/gitbook/historical-search/panther-fields.md]: Switched to table .

File: docs/gitbook/historical-search/panther-fields.md (hunk 6 above)
Comment (rleighton) [reply docs/gitbook/historical-search/panther-fields.md]: I copied this style from the README.md ... fixed here and there.

File: docs/gitbook/SUMMARY.md (hunk 7 above)
Comment (rleighton) [reply docs/gitbook/SUMMARY.md]: Ah, did not  catch that. Thx, fixed.

File: docs/gitbook/historical-search/panther-fields.md (hunk 8 above)
Comment (rleighton) [reply docs/gitbook/historical-search/panther-fields.md]: Yes!. Fixed.

File: docs/gitbook/historical-search/panther-fields.md (hunk 2 above)
Comment (rleighton) [reply docs/gitbook/historical-search/panther-fields.md]: Thx, fixed

**Pull request merged**